.cargo/config

```diff
@@ -1,2 +1,15 @@
 [alias]
 regen-protobufs = ["run", "--bin", "protobuf-gen", "tools/protobuf_files.toml"]
+dev-install = ["install", "asdev"]
+all_tests = ["asdev", "test"]
+rust_tests = ["asdev", "test_rust"]
+verify_desktop = ["asdev", "verify_desktop"]
+verify_android = ["asdev", "verify_android"]
+verify_ios = ["asdev", "verify_ios"]
```

Comment:
@tarikeshaq hm looking at this list of `verify`. What do you think about squashing this into something like `cargo verify_env` (`cargo verify` might be too generic)? Then we can leave the other scripts alone in https://github.com/mozilla/application-services/pull/3218/files#r438921631 but prefix that part of the documentation to just run `cargo verify_env` and get output for all 3 platforms

Reply:
That's possible! I'd have to change `asdev` to support doing all three back to back, but that's easy enough. Was wondering, would that be better, or maybe add an 'optional' section and leave the other documentation alone. Something like:
## (Optional) Dev tool
You can also install and use `asdev` to help you run the above commands, you can do that by running the following:

```
$ cargo dev-install
```

This will allow your to run the above commands without having to remember where the scripts are. For example, you would be able to run `cargo verify_desktop` which would run `./libs/verify-desktop-environment.sh`. To check the available commands, run `cargo asdev -h` or open the dialog using `cargo asdev`

Reply:
> I'd have to change asdev to support doing all three back to back, but that's easy enough. 

Yeap, let's do that 👍